### DISCUSSION TITLE
Welcome to Noriben Discussions!

### DISCUSSION THREAD

**Discussion by Rurik** [Rurik/Noriben#43] [General]
## Welcome!
  I've enabled Discussions as part of this project for questions, ideas, and sharing. 

Please feel free to make a new discussion thread for any topic that may not be a direct Issue.

**Comment by ZeroX29a** [comment]
Is there any way to log as the the change is happening, as in my case petya force the shutdown, but will only encrypt the files after rebooting, which I can retrieve using another distro.

**Reply by Rurik** [reply]
In this case, not really. You could manually start the encryption after the reboot, and after launching Noriben. 

It may be possible to create a shortcut to Noriben from a startup location and have it immediately launch, but there is no guarantee that it will launch before the ransomware does.
